app/services/invoices/payments/gocardless_service.rb

```diff
@@ -5,6 +5,18 @@ module Payments
     class GocardlessService < BaseService
       include Customers::PaymentProviderFinder
 
+      class MandateNotFoundError < StandardError
+        DEFAULT_MESSAGE = 'No mandate available for payment'
+
+        attr_reader :code
+
+        def initialize(msg = DEFAULT_MESSAGE)
+          @code = '999'
```

Comment:
We can use something like `No mandate error` or `no_mandate_error`.

Reply:
looks good